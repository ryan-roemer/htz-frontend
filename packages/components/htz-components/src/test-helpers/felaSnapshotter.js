import React from 'react';
import createRenderer from '../components/StyleProvider/createRenderer';
import StyleProvider from '../components/StyleProvider/StyleProvider';
import createFelaSnapshotter from './createFelaSnapshotter';

const testRenderer = createRenderer({ isRtl: true, });
const felaSnapshotter = createFelaSnapshotter(StyleProvider, testRenderer);

export default felaSnapshotter;
