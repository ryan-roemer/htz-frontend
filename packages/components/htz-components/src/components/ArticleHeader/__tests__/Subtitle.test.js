import React from 'react';
import felaSnapshotter from '../../../test-helpers/felaSnapshotter';
import Subtitle from '../_Subtitle'; // eslint-disable-line import/no-named-as-default

describe('Kicker component', () => {
  it('renders correctly a \'p\' tag', () => {
    const snapshot = felaSnapshotter(
      <Subtitle>ישראלים בצפון, אם נתניהו יחליט שאלה פעולות נדרשות</Subtitle>
    );
    expect(snapshot).toMatchSnapshot();
  });
});
