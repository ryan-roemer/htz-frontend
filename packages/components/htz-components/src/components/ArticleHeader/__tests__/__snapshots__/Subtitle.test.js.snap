// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kicker component renders correctly a 'p' tag 1`] = `
Object {
  "component": <div>
    <p
      className="htz-a htz-b htz-c htz-d htz-e"
    >
      ישראלים בצפון, אם נתניהו יחליט שאלה פעולות נדרשות
    </p>
  </div>,
  "styles": ".htz-a{margin-top:2rem}@media (max-width: 80em){.htz-b{font-size:2.6666666666666665rem}.htz-c{line-height:1.5em}}@media (min-width: 80em){.htz-d{font-size:3rem}.htz-e{line-height:1.3333333333333333em}}",
}
`;
